README.md

```diff
@@ -11,7 +11,7 @@ Solid Queue can be used with SQL databases such as MySQL, PostgreSQL or SQLite,
 Solid Queue is configured by default in new Rails 8 applications. But if you're running an earlier version, you can add it manually following these steps:
 
 1. `bundle add solid_queue`
-2. `bin/rails solid_queue:install`
```

Comment:
I believe this bit is correct for the current version (following https://github.com/rails/solid_queue/commit/564047dbb9813db7b77101bf45cace021b15ac01, see https://github.com/rails/solid_queue/issues/363)